src/RefactorHelper.cs

```diff
@@ -21,6 +21,7 @@
 using System.Linq;
 using Gauge.Dotnet.Extensions;
 using Gauge.Dotnet.Models;
+using Gauge.Messages;
```

Comment:
In general, I don't like proto messages going this deep into runners. I suggest we have runners' own models for this diff and translate them while sending the response. this will reduce the coupling between proto and runner.